singleheader/amalgamate.sh

```diff
@@ -127,8 +127,8 @@ echo "/* auto-generated on ${timestamp}. Do not edit! */" > ${AMAL_C}
     done
 } >> ${AMAL_C}
 
-cp -u "${SCRIPTPATH}/amalgamate_demo.cpp" "${DEMOCPP}" 2>/dev/null
-cp -u "${SCRIPTPATH}/README.md" "${README}" 2>/dev/null
+cp -u "${SCRIPTPATH}/amalgamate_demo.cpp" "${DEMOCPP}" 2>/dev/null || :
```

Comment:
What does the `:` do? This is new syntax to me.

Will cp -f do what we want? I don't care which, as long as this works everywhere.

Reply:
Also, if the source file isn't there and it therefore fails to copy, will we still get an error code?

Reply:
The "-u" flag is GNU-specific. It is likely not to work under freeBSD and macos.